src/components/FilterChecklist.vue

```diff
@@ -2,9 +2,8 @@
   <fieldset>
     <div class="filters-title" @click.prevent="toggleFilterVisibility">
       <span>{{ title }}</span>
-
-      <button class="filter-dropdown-visibility-button">
-        <img src="@/assets/arrow-bottom.svg" class="arrow-bottom" alt="toggle filters visibility" />
+  <button v-bind:class="[filtersVisible ? 'rotImg' : 'filter-dropdown-visibility-button']">
```

Comment:
It's better to use something like this instead to toggle the `rotImg` class https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax